lib/ioki/apis/operator_api.rb

```diff
@@ -401,6 +401,12 @@ class OperatorApi
         base_path:   [API_BASE_PATH, 'products', :id],
         model_class: Ioki::Model::Operator::RestrictedArea,
         except:      [:update]
+      ),
+      Endpoints.crud_endpoints(
+        :driver_multicast,
+        base_path:   [API_BASE_PATH, 'products', :id],
+        model_class: Ioki::Model::Operator::DriverMulticast,
+        except:      [:update, :delete]
```

Comment:
I believe you could also use `only`, if that's a bit easier. Maybe a note for the future, as we're using mostly `except`, but that's more error prone than an allowlist :slightly_smiling_face:

Reply:
True and good point 👍